src/components/entity/filter/EntityFilterDialog.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <v-dialog
+    v-model="dialog"
+    fullscreen
+    hide-overlay
+    transition="dialog-bottom-transition"
+  >
+    <template v-slot:activator="{ on, attrs }">
+      <a
+        @click="dialog = true"
+        :class="{ disabled: loading }"
+        :aria-disabled="loading"
+      >
+        <v-icon
+          dense
+          v-bind="attrs"
+          v-on="on"
+          :color="loading ? 'gray' : 'black'"
+          >{{ mdiFilterOutline }}</v-icon
+        >
+        Filters
+      </a>
+    </template>
+    <v-card>
+      <v-toolbar flat color="white">
+        <v-toolbar-title class="pl-1">Filter results</v-toolbar-title>
+        <v-spacer></v-spacer>
+        <v-toolbar-items class="mr-n2">
+          <v-btn icon color="black" @click="dialog = false">
+            <v-icon>{{ mdiClose }}</v-icon>
+          </v-btn>
+        </v-toolbar-items>
+      </v-toolbar>
+      <EntityFilterFacetPanels class="filter-dialog" />
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+import { mapGetters } from "vuex";
+import EntityFilterFacetPanels from "@/components/entity/filter/EntityFilterFacetPanels.vue";
+
+interface Data {
+  dialog: boolean;
+}
+
+@Component({
+  components: {
+    EntityFilterFacetPanels,
+  },
+  computed: {
+    ...mapGetters(["loading", "mdiFilterOutline", "mdiClose"]),
+  },
+})
+export default class EntityFilterDialog extends Vue {
+  data(): Data {
+    return {
+      dialog: false,
+    };
+  }
+}
+</script>
+
+<style scoped>
```

Comment:
Scope styles with SASS. There is already a color variable for `gray` and `white` (`$gray` and `$white`)